src/parameters/cmp_p3.jl

```diff
@@ -0,0 +1,31 @@
+"""
+    CloudMicrophysicsParametersP3{FT}
+
+Parameters for P3 bulk microphysics scheme
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct CloudMicrophysicsParametersP3{FT} <: AbstractCloudMicrophysicsParameters
+    "Coefficient in mass(size) relation [g/μm^β_va]"
+    α_va::FT
+    "Coefficient in mass(size) relation [-]"
+    β_va::FT
+    "Cloud ice density [kg/m3]"
+    ρ_i::FT
+    "Cloud liquid water density [kg/m3]"
+    ρ_l::FT
+    "Coefficient in area(size) for ice side plane, column, bullet, and planar polycrystal aggregates from Mitchell 1996 [μm^(2-σ)]"
+    γ::FT
+    "Coefficient in area(size) for ice side plane, column, bullet, and planar polycrystal aggregates from Mitchell 1996 [-]"
+    σ::FT
+end
+
+CloudMicrophysicsParametersP3(param_set) = CloudMicrophysicsParametersP3(
```

Comment:
This looks good. But it is best to contruct this directly from the `ClimaParameters` dictionary/ toml file! This way, we can also keep deleting fields from the bigger `CloudMicrophysicsParameters` struct as we keep moving fields to smaller structs!

Reply:
Lets leave it for now, so that we don't do any breaking changes for ClimaAtmos and other upstream packages. We will delete the big struct once we are done with reorganizing things and make another breaking release then.